docs/framework/data/adonet/sql/sqlclient-support-for-high-availability-disaster-recovery.md

```diff
@@ -36,6 +36,9 @@ This topic discusses SqlClient support (added in [!INCLUDE[net_v45](../../../../
 1.  <xref:System.Data.SqlClient.SqlConnectionStringBuilder.ApplicationIntent%2A>  
   
 2.  <xref:System.Data.SqlClient.SqlConnectionStringBuilder.MultiSubnetFailover%2A>  
+
+> [!NOTE]
+>  Specifying `MultiSubnetFailover` is not required with [!INCLUDE[net_v461](../../../../../includes/net-v461-md.md)]) or later.
```

Comment:
Wondering if it'd be better to say similar to the blog post text:
Setting `MultiSubnetFailover` to `true` isn't required with ...

or later. -> or later versions.